napster/meson.build

```
@@ -0,0 +1,17 @@
+feature = get_option('napster')
+
+summary({'Napster': feature}, section : 'Protocols')
+
+if not feature.allowed()
+  subdir_done()
+endif
+
+novell_prpl = shared_library('napster',
```

Maybe we should remove these variables since they don't get used, but...
```suggestion
napster_prpl = shared_library('napster',
```